src/components/Section.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import {View} from 'react-native';
 import PropTypes from 'prop-types';
+import _ from 'underscore';
```

Comment:
Should we use the lodash here?

Reply:
@marcaaron 

I have no idea, what do you think @grgia ?

Reply:
In TS files, don’t use underscore at all, and only use lodash for functions that don’t have a 1:1 vanilla JS equivalent